### DISCUSSION TITLE
Erratic behaviour of Faikin

### DISCUSSION THREAD

**Discussion by alexiusMTB** [revk/xESP32-Faikout#407] [Q&A]
Hi all, my Faikin was working perfectly for a few months now but since last week it's starting to behave erratic. It switches back to Auto mode all the time and also the target temperature switches all by it self.

**Comment by revk** [comment]
That is weird. Can you get logs of MQTT messages?

**Reply by alexiusMTB** [reply]
That's another problem, I also can't get connection in Home assistant (via MQTT)
The faikin Browser environment works but as mentioned before the Faikin behaves strange. If I switch to night mode it runs night for a few minutes and then switches back to Auto and seems to try to get to the target temperature quite aggressive. If the target is reached it switches of the AC and the led and the app even indicate that heating mode is on!

**Reply by revk** [reply]
OK, night/auto is a known issue with S21 and in the manual - there are various settings to change how that works.

If using Faikin auto mode - the temperature is set high/low to get to that. Faikin auto mode is also documented in the manual and as lot of settings to fine tune how it works.
